t/091-coroutine.t

```diff
@@ -483,8 +483,8 @@ done
 GET /lua
 --- response_body
 hello
---- error_log eval
-["stack traceback:", "coroutine 0:", "coroutine 1:", "coroutine 2:"]
+--- no_error_log
+[error]
```

Comment:
Also, better check the return values of `coroutine.resume` for better test coverage.